web/src/utils/exporter.js

```diff
@@ -65,6 +66,36 @@ export function svg2url(svgElement, options = {}) {
     copy.insertAdjacentHTML('afterbegin', `<style>${rules.join('\n')}</style>`);
   }
 
+  // Need some special treatment for C3 charts.
```

Comment:
can you please extract this custom logic to a separate function

Reply:
Let me know if edc63cd looks ok.